### PR TITLE
Add information about installing nix by pacman

### LINKED ISSUE #398: [advice] Add install method to <https://nix.dev/tutorials/install-nix#linux>
In archlinux, we can `sudo pacman -S nix` to install nix. Why not add this information to <https://nix.dev/tutorials/install-nix#linux>?

### PR DESCRIPTION
**Pull request by Freed-Wu**
Fix #398

### CODE REVIEW

**Comment by Freed-Wu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by fricklerhandwerk** [comment]
@domenkozar @lucperkins @bryanhonof @thufschmitt Do we want to support external installation options?

**Comment by thufschmitt** [comment]
> @domenkozar @lucperkins @bryanhonof @thufschmitt Do we want to support external installation options?

I think it's great to mention them, but it should be explicit that these are 3d-party packages and not official options

File: source/tutorials/install-nix.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ $ sh <(curl -L https://nixos.org/nix/install) --daemon
 For security you may want to [verify the installation script] using GPG signatures.
 :::
 
+For Archlinux, you can install nix by `pacman`. See <https://wiki.archlinux.org/title/Nix>.
```
Comment (fricklerhandwerk) [inline source/tutorials/install-nix.md:17]:
```suggestion
## Arch Linux

On Arch Linux, you can alternatively [install Nix through `pacman`](https://wiki.archlinux.org/title/Nix#Installation).
```

**Comment by Freed-Wu** [comment]
Fixed.

**Comment by fricklerhandwerk** [comment]
Thanks, LGTM. Let's wait for a bit more feedback. Please ping me again if you feel it has timed out. Maybe a week? People are busy.

**Comment by Freed-Wu** [comment]
> Maybe a week? People are busy.

Can understand :+1:

**Comment by fricklerhandwerk** [comment]
@yukiisbored I resolved conflicts to accommodate the new tab view. Could you please take a look, does it make sense to you that way?

**Review by fricklerhandwerk — APPROVED** [review]

**Review by yukiisbored — APPROVED** [review]

**Comment by yukiisbored** [comment]
@fricklerhandwerk LGTM

**Comment by yukiisbored** [comment]
Seems like the CI requires a retry since it seems like a fluke (502 bad request to nixos.org)

**Pull request merged**